kafka/cluster.py

```diff
@@ -119,6 +120,7 @@ def partitions_for_broker(self, broker_id):
 
         Returns:
             set: {TopicPartition, ...}
+            None if the broker either has no partitions or does not exist.
         """
         return self._broker_partitions.get(broker_id)
```

Comment:
This was surprising behavior to me, I expected a broker that exists but has no partitions would return an empty set. So I'm not sure if it's better to document the current behavior, or change this to `get(broker_id, set())`.